tests/console/yast2_ntpclient.pm

```diff
@@ -14,14 +14,40 @@ use strict;
 use base "console_yasttest";
 use testapi;
 use utils qw(type_string_slow zypper_call);
+use version_utils qw(is_sle sle_version_at_least is_leap leap_version_at_least);
 
 sub run {
     select_console 'root-console';
-    # Test often fails due to info kernel messages disrupting screen
-    # Decrease logging level to warning to avoid this
-    assert_script_run "dmesg -n 4";
+
+    # ntp configuration is different in SLE15/Leap15/Tumbleweed
+    # use sle_or_leap_15 variable to avoid executing is_* and *_version_at_least multiple time
+    my $sle_or_leap_at_least_15 = ((is_sle && sle_version_at_least '15') || (is_leap && leap_version_at_least '15.0' || is_tumbleweed)) ? 1 : 0;
```

Comment:
We enable same behavior for TW here, because of the way how brackets are set, I'm not sure if it's intended. As of now this always will be true in case of TW, which is not corresponding to the variable name.

Reply:
Your're right, the variable name is not good now. I added TW after.
I think that TW use Chrony now, so I will rename the variable.

Reply:
I am not so sure about "is_tumbleweed": It is sometimes used as a distribution check, sometimes version check and if it would be for me I would try to avoid it at all because tumbleweed should be our default which never appears anywhere. Instead just specify the exceptions, e.g. !is_sle

Reply:
Thx, I will modify this.

Reply:
TW doesn't use Chrony yet, so I will do the test like this: `((is_sle && sle_version_at_least '15') || (is_leap && leap_version_at_least '15.0'))`, because I need to check the Leap version, and I think it's easier to read.

Maybe `is_tumbleweed` and `is_leap` should be removed? But this is a much more complicated task :)